### PR TITLE
Fix CLI install script to work in OneVerily envs.

### PR DESCRIPTION
**Pull request by jmczerk**

### CODE REVIEW

File: startupscript/install-cli.sh [hunk 1]
```diff
@@ -22,6 +22,20 @@ set -o xtrace
 source "${SCRIPT_DIR}/emit.sh"
 source "${CLOUD_SCRIPT_DIR}/vm-metadata.sh"
 
+get_axon_version_url() {
```
Comment (mbookman) [inline startupscript/install-cli.sh:25]:
```suggestion
function get_axon_version_url() {
```

File: startupscript/install-cli.sh [hunk 2]
```diff
@@ -22,6 +22,20 @@ set -o xtrace
...
+    "verily-devel") echo "https://terra-devel-axon.api.verily.com/version" ;;
+    "verily-autopush") echo "https://terra-autopush-axon.api.verily.com/version" ;;
+    "verily-staging") echo "https://terra-staging-axon.api.verily.com/version" ;;
+    "verily-preprod") echo "https://terra-preprod-axon.api.verily.com/version" ;;
+    "dev-stable") echo "https://workbench-dev.verily.com/api/axon/version" ;;
+    "dev-unstable") echo "https://workbench-dev-unstable.verily.com/api/axon/version" ;;
+    "test") echo "https://workbench-test.verily.com/api/axon/version" ;;
+    *) echo echo "ERROR: ${TERRA_SERVER} is not a known Workbench server"; exit 1 ;;
+  esac
+}
```
Comment (mbookman) [inline startupscript/install-cli.sh:37]:
```suggestion
}
readonly -f get_axon_version_url
```

File: startupscript/install-cli.sh [hunk 3]
```diff
@@ -33,21 +47,16 @@ readonly TERRA_SERVER
...
-      exit 1
-    fi
-    cliDistributionPath="$(echo "${versionJson}" | jq -r '.cliDistributionPath')"
-
-    ${RUN_AS_LOGIN_USER} "curl -L https://storage.googleapis.com/${cliDistributionPath#gs://}/download-install.sh | TERRA_CLI_SERVER=${TERRA_SERVER} bash"
-    cp wb "${WORKBENCH_INSTALL_PATH}"
-  else
-    >&2 echo "ERROR: ${TERRA_SERVER} is not a known Workbench server"
+  # Map the CLI server to appropriate AFS service path and fetch the CLI distribution path
+  if ! versionJson="$(curl -s "$(get_axon_version_url "${TERRA_SERVER}")")"; then
```
Comment (mbookman) [inline startupscript/install-cli.sh:51]: We've had a convention in the scripts of UPPER_CASE variables being for script global and lower_case variables being function local variables. Is that intentionally not being preserved here or could we dpo that?

**Review by mbookman — CHANGES_REQUESTED** [review]

**Comment by jmczerk** [comment]
@mbookman please give this another look when you get a chance, as this is blocking OneVerily tests.

**Review by yuhuyoyo — APPROVED** [review]

**Review by cbookg — APPROVED** [review]

**Review by mbookman — APPROVED** [review]
LGTM!

**Pull request merged**